### ISSUE TITLE
Vagrant on GCE

### ISSUE THREAD

**Issue by st-isidore-de-seville** [test-kitchen/kitchen-vagrant#188]
Hello, I am having an issue where if I am running kitchen-vagrant on GCE, I am getting an error that states: 

```
STDERR: Timed out while waiting for the machine to boot. This means that
  Vagrant was unable to communicate with the guest machine within
  the configured ("config.vm.boot_timeout" value) time period.
```

There does not appear to be a way to set this using kitchen-vagrant.  

Is there a way to support it?

Thanks!

**Comment by cheeseplus** [comment]
This is potentially something we could support but is there a reason you aren't using https://github.com/test-kitchen/kitchen-google or similar? Generally speaking we only support local boxes with the vagrant plugin as each cloud has it's own respective one.

**Comment by st-isidore-de-seville** [comment]
hey @cheeseplus!  At the time, I was testing on Travis CI which was using GCE.  Instead of vagrant, I could spin up a cloud instance but I tried avoiding doing so just to avoid the cost so I figured I would try vagrant on GCE since people at the time appeared to have luck with it.  However since then, I have been embracing kitchen-docker & kitchen-dokken more.  I think this would be an interesting feature if I could get to to work but not high priority at this point.
